src/components/items/Event.jsx

```diff
@@ -130,7 +130,7 @@ function Event(props) {
           {ishost ? (
             <NavItem>
               <NavLink
-                className={{ active: activeTab === "2" }}
+                /* className={{ active: activeTab === "2" }} */
```

Comment:
unsure of why this is commented out.

Reply:
These are invalid and were generating an error on the console.log className has to be a string.